src/urbit/sur/chess.hoon

```diff
@@ -153,6 +151,8 @@
       [%offer-draw game-id=@dau]
       [%accept-draw game-id=@dau]
       [%decline-draw game-id=@dau]
+      [%change-special-draw-preference game-id=@dau setting=?]
+      [%claim-special-draw game-id=@dau]
```

Comment:
As above, might want to change.

Reply:
See above - this was my call for merging threefold repetition and the 50-move rule into one logic flow